wallets/core/src/types.ts

```diff
@@ -79,6 +83,12 @@ export type SwitchNetwork = (options: {
   newInstance?: TryGetInstance;
 }) => Promise<void>;
 
+export type SuggestChain = (options: {
```

Comment:
Please rename all the `SuggestChain`s to `suggestAndConnect`

Reply:
Done